mini_data/routing/attributes/attributes_lv05/attributes.csv

```diff
@@ -0,0 +1,6 @@
+HYBAS_ID,area,DIST_MAIN
```

Comment:
What units is DIST_MAIN in? are we sure it is in m? these basins are all only ~1km apart then, which seems small, but these may be very small basins

Reply:
nvm! I see that we convert from km to  m internally.